pkg/tunneldriver/driver.go

```diff
@@ -223,8 +223,12 @@ func handleConn(ctx context.Context, dest string, protocol string, dialer Dialer
 
 	// Support HTTPS backends
 	if protocol == "HTTPS" {
+		host, _, err := net.SplitHostPort(dest)
+		if err != nil {
+			return err
+		}
```

Comment:
This will return an error if no port is found: https://go.dev/play/p/vSHWL0VeJxV

Probably better to do:
```suggestion
		host, _, err := net.SplitHostPort(dest)
		if err != nil {
			host = dest
		}
```